src/blueair_api/ir_aws.py

```diff
@@ -0,0 +1,176 @@
+from typing import Any, TypeVar
```

Comment:
let's spell it out instead of ir, intermediate_representation_aws.py

Reply:
I see. From where I came from IR is ubiquitous  for intermediate-representation (e.g. [MLIR](https://mlir.llvm.org/docs/LangRef/), but I recognize that IR is probably more often used for Infra-red in home automation (and impulse response in audio systems.

Should we rename this to something that's less confusing for our context? Like [HAL](https://en.wikipedia.org/wiki/HAL_(software), or maybe just call it metadata_aws.py?

Reply:
I'm happy to expand people's vocabulary, I'd just rather be sure it is clear without abbreviation

Reply:
Sure. We will stick with intermediate_representation then. (I still used import ... as ir to simplify the namespacing of its methods).